presto-native-execution/CMakeLists.txt

```diff
@@ -57,6 +57,8 @@ option(PRESTO_ENABLE_TESTING "Enable tests" ON)
 
 option(PRESTO_ENABLE_JWT "Enable JWT (JSON Web Token) authentication" OFF)
 
+option(PRESTO_ENABLE_LINUX_MEMORY_CHECKER "Enable LinuxMemoryChecker" OFF)
```

Comment:
Can we add this to the Makefile as well (see PRESTO_ENABLE_JWT example)?

In addition, we should document this in `presto-docs/src/main/sphinx/presto_cpp/features.rst` with the new options, a basic intro and how it is enabled at build.

Reply:
Added to Makefile:
```
ifeq ($(PRESTO_ENABLE_LINUX_MEMORY_CHECKER), ON)
  EXTRA_CMAKE_FLAGS += -DPRESTO_ENABLE_LINUX_MEMORY_CHECKER=ON
endif
```